app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy(command: str) -> None:
+    cmd, current, future = command.split()
```

Comment:
It still will be much more descriptive to name variables like `current_file`

Reply:
@yakhlamova I meant the same for the `future` variable. You should name it more descriptively

Reply:
I tried, but then I have a problem with the line too long, so I was in search of a nice name for this variable but I can`t pick up so I left what it was

Reply:
`cmd, current_file, future_file = command.split()`
`with (open(current_file, "r") as file_in,
              open(future_file, "w") as file_out):`